src/js/controllers/chatController.js

```diff
@@ -147,49 +135,27 @@ module.exports = ViewController.extend({
     },
 
     _receiveMessage: function(message) {
-        if (!!this.conversation) {
+        return this._getConversation().then(function(conversation) {
 
             // we actually need to extract the appMakers first
             // since the message rendering is done on message add event
             // and some UI stuff is relying on the appMakers collection
-            if (!this.conversation.get('appMakers').get(message.authorId)) {
-                this.conversation.get('appMakers').add({
+            if (!conversation.get('appMakers').get(message.authorId)) {
+                conversation.get('appMakers').add({
```

Comment:
I guess we don't need that logic either since I can just rely on the role to know if it's an appMaker